backend/apps/ai/management/commands/ai_create_project_chunks.py

```
@@ -0,0 +1,225 @@
+        except openai.OpenAIError as e:
+            self.stdout.write(self.style.ERROR(f"OpenAI API error for project {project.key}: {e}"))
+            return []
+
+    def extract_project_content(self, project: Project) -> tuple[str, str]:
+        prose_parts = []
+        metadata_parts = []
+
+        if project.name:
+            metadata_parts.append(f"Project Name: {project.name}")
+
+        if project.description :
```

Flake8 (7.2.0)

[error] whitespace before ':'

(E203)